src/ds/d500/d500-device.cpp

```diff
@@ -719,17 +719,31 @@ namespace librealsense
 
     bool d500_device::check_symmetrization_enabled() const
     {
-        using namespace ds;
-        command cmd(GET_HKR_CONFIG_TABLE,
-            static_cast<int>(d500_calib_location::d500_calib_ram_memory),
-            static_cast<int>(d500_calibration_table_id::stream_pipe_config_id),
-            static_cast<int>(d500_calib_type::d500_calib_dynamic));
-        auto res = _hw_monitor->send(cmd);
+        // The following try catch block has been added to avoid
+        // device's constructor failure for users working with new librealsense
+        // version and old fw version (which would not have the stream pipe config table)
+        // Only the content of the try statemewnts should be kept, after some time.
```

Comment:
Small typo statemewnts

Reply:
Fixed, beside that anything else?

Reply:
No further issues.
This restores the old code that worked with older FW versions.